File: Flutter-Harpia-Health-Analysis/lib/model/firebase/FcmData.dart
class FcmData {
  late String _url;
  late String _msgTitle;
  late String _msg;
  late bool _showNotification;
  late int _patientId;

  FcmData(
      {url = "",
      required msgTitle,
      required msg,
      required showNotification,
      required int patientId}) {
    _url = url;
    _msg = msg;
    _msgTitle = msgTitle;
    _showNotification = showNotification;
    _patientId = patientId;
  }

  factory FcmData.fromJson(Map<String, dynamic> json) {
    bool showNotification = bool.parse(json["showNotification"] as String);
    int patientId=int.parse(json["patientId"] as String);
    return FcmData(
      msgTitle: json["msgTitle"] as String,
      msg: json["msg"] as String,
      url: json["url"] as String,
      patientId:patientId,
      showNotification: showNotification,
    );
  }

  @override
  String toString() {
    return 'FcmData{_url: $_url, _msgTitle: $_msgTitle, _msg: $_msg, _showNotification: $_showNotification, _patientId: $_patientId}';
  }

  String get msg => _msg;

  String get msgTitle => _msgTitle;

  String get url => _url;

  bool get showNotification => _showNotification;

  int get patientId => _patientId;
}

File: Harpia-Health-Analysis-WS/src/main/java/com/harpia/HarpiaHealthAnalysisWS/model/firebase/FcmData.java
package com.harpia.HarpiaHealthAnalysisWS.model.firebase;

public class FcmData {
    String url = "";
    String msgTitle = "";
    String msg = "";
    String reasonSend = "";
    int reasonCode;
    boolean showNotification = false;
    long patientId;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

    public String getMsgTitle() {
        return msgTitle;
    }

    public void setMsgTitle(String msgTitle) {
        this.msgTitle = msgTitle;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }

    public boolean isShowNotification() {
        return showNotification;
    }

    public void setShowNotification(boolean showNotification) {
        this.showNotification = showNotification;
    }

    public long getPatientId() {
        return patientId;
    }

    public void setPatientId(long patientId) {
        this.patientId = patientId;
    }

    public String getReasonSend() {
        return reasonSend;
    }

    public void setReasonSend(String reasonSend) {
        this.reasonSend = reasonSend;
    }

    public int getReasonCode() {
        return reasonCode;
    }

    public void setReasonCode(int reasonCode) {
        this.reasonCode = reasonCode;
    }

    @Override
    public String toString() {
        return "FcmData{" +
                "url='" + url + '\'' +
                ", msgTitle='" + msgTitle + '\'' +
                ", msg='" + msg + '\'' +
                ", reasonSend='" + reasonSend + '\'' +
                ", reasonCode=" + reasonCode +
                ", showNotification=" + showNotification +
                ", patientId=" + patientId +
                '}';
    }
}
